lib/src/builder/build_step_impl.dart

```diff
@@ -5,16 +5,24 @@ import 'dart:async';
 import 'dart:collection';
 import 'dart:convert';
 
+import 'package:code_transformers/resolver.dart' as code_transformers;
+
+import '../analyzer/resolver.dart';
 import '../asset/asset.dart';
 import '../asset/id.dart';
 import '../asset/reader.dart';
 import '../asset/writer.dart';
+import '../util/barback.dart';
 import 'build_step.dart';
 import 'exceptions.dart';
 
 /// A single step in the build processes. This represents a single input and
 /// its expected and real outputs. It also handles tracking of dependencies.
 class BuildStepImpl implements BuildStep {
+  static code_transformers.Resolvers _resolvers =
+      new code_transformers.Resolvers.fromMock(
```

Comment:
Will switch this to use the actual dart sdk until we figure out how we want to make it configurable.